waspc/src/Wasp/Generator/SdkGenerator.hs

```diff
@@ -188,20 +188,21 @@ npmDepsForSdk spec =
             ("mitt", "3.0.0"),
             ("react", "^18.2.0"),
             ("lodash.merge", "^4.6.2"),
-            ("react-router-dom", "^5.3.3"),
+            ("react-router-dom", show reactRouterVersion),
             ("react-hook-form", "^7.45.4"),
             ("superjson", "^1.12.2"),
             -- Todo: why is this in dependencies, should it be in dev dependencies?
             -- Should it go into their package.json
             ("@types/express-serve-static-core", "^4.17.13"),
-            ("@types/react-router-dom", "^5.3.3")
+            ("@types/react-router-dom", show reactRouterVersion)
           ]
           ++ depsRequiredForAuth spec
           ++ depsRequiredByOAuth spec
-          -- This must be installed in the SDK because it lists prisma/client as a dependency.
+          -- Server auth deps must be installed in the SDK because "@lucia-auth/adapter-prisma"
```

Comment:
Updated this comment since I talked with @sodic recently twice and always forgot what was it really talking about.